chain-signatures/node/src/protocol/presignature.rs

```diff
@@ -23,12 +27,94 @@ use near_account_id::AccountId;
 pub type PresignatureId = u64;
 
 /// A completed presignature.
+#[derive(Clone)]
 pub struct Presignature {
```

Comment:
`Presignature` should never be clone'able since that introduces reusage of presignature. Why do we even need it at all?

Reply:
yeah, for testing, ok security first